app/src/main/res/values-en/strings.xml

```diff
@@ -719,12 +719,12 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_buildingType_bungalow_description2">small detached house (summer house, holiday cottage, cabin …)</string>
     <string name="quest_buildingType_static_caravan">Mobile home</string>
     <string name="quest_buildingType_commercial">Commercial building</string>
-    <string name="quest_buildingType_commercial_generic_description">building where people work, shop or do any other commercial activities</string>
+    <string name="quest_buildingType_commercial_generic_description">building where people work, shop or do other commercial activities</string>
     <string name="quest_buildingType_industrial">Industrial building</string>
     <string name="quest_buildingType_industrial_description">e.g. a factory, workshop, car repair garage, …</string>
     <string name="quest_buildingType_office">Office building</string>
     <string name="quest_buildingType_retail">Shop(s)</string>
```

Comment:
The “(s)” construction is rather formal in English. Can we get away with just saying “Shops”? I think people would associate this option with a single standalone retail store.

Also, this should say “Store”, rather than “Shop”, for consistency with the description.

Reply:
I think that it would confusing at least to me

Reply:
Maybe just "retail building" would be fine too? After all, that's the tag - `building=retail`.

Though, at least in German, "retail" means "a shop aimed to sell things to the end-consumer", in contrast to e.g. wholesale shops. But `building=retail` encompasses any building that houses a commercial activity aimed at the end consumer, so including restaurants, cafés and any other kinds of places that would rent a generic vacant shop space.

"Shop(s)" is somewhat lent from the German "Laden", which (can but uncommonly) also describe the generic **space** for a shop ("Geschäftsraum") and not the shop per se.
Maybe there is an English word that captures that?

Reply:
“Retail building” would work too. American English makes more or less the same retail/commercial distinction as `building=retail/commercial`, although people often call both “commercial” in the sense of having commercial business activity. I’d be more confident about using the term “retail” for a quest about building classification than about land use classification, which is where mappers seem to munge the distinction most often.

Reply:
Let's use retail building then...

```suggestion
    <string name="quest_buildingType_retail">Retail building</string>
```